README.md

```diff
@@ -479,6 +479,48 @@ Contributions of additional uses cases on top of ECS are welcome.
 * *Use prefixes.* Fields must be prefixed except for the base fields. For example all `host` fields are prefixed with `host.`. See `dot` notation in FAQ for more details.
 * Do not use abbreviations. (A few exceptions like `ip` exist.)
 
+## Understanding ECS conventions
+
+### Multi-fields text indexing
+
+ElasticSearch can index text multiple ways:
+
+* `text` indexing allows for full text search, or searching arbitrary words that
+  are part of the field.
+* `keyword` indexing allows for exact match search (much faster) and allows for
```

Comment:
And prefix which is pretty nice for version fields for example: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-prefix-query.html